New version: ControlSystemIdentification v2.11.5

JuliaRegistrator opened the pull request:
- Registering package: ControlSystemIdentification
- Repository: https://github.com/baggepinnen/ControlSystemIdentification.jl
- Created by: @baggepinnen
- Version: v2.11.5
- Commit: 84282a26aefe3cafab34458b1b1bb954a3fe3dc2
- Reviewed by: @baggepinnen
- Reference: https://github.com/baggepinnen/ControlSystemIdentification.jl/issues/5#issuecomment-2655476000
- Description: System Identification toolbox, compatible with ControlSystems.jl
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359810c508f76bae95c919d8dcfec7a1fa7b9bded3db48cf3c5d6a5f032d281c131d8dd0d7b02e0e775ea3e5879f562354cfffb4f2b49c8607f8a5729624ec2bbb6fc23aeb978eaf840f7221dd0f200c4e086a2dc8517e4fb2aa6489d21418702987bf53f39ec149862f8821e551634be92e27202b7404b40d53afd10052e4adaeafdb622a85521ec23ae3709504e78381b6 -->

baggepinnen commented:
This can be closed in favor of https://github.com/JuliaRegistries/General/pull/124704

JuliaRegistrator committed:
New version: ControlSystemIdentification v2.11.5

UUID: 3abffc1c-5106-53b7-b354-a47bfc086282
Repo: https://github.com/baggepinnen/ControlSystemIdentification.jl.git
Tree: 53fcea6af8fd2632622cfc07d73f4b95ca3b592a

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
